### ISSUE TITLE
As an SRAA I should be able to switch Institution within the RA interface

### ISSUE THREAD

**Issue by phavekes** [OpenConext/Stepup-Project#80]
[This issue is imported from pivotal - Originaly created at Feb 2, 2015 by Daan van Renterghem](https://www.pivotaltracker.com/story/show/87413848)

Implement the Institution switcher for SRAA\'s in the RA application UI, including the reloading of the page after switching the Identity

Blocked by https://www.pivotaltracker.com/story/show/87412486
Blocked by https://www.pivotaltracker.com/story/show/87412402

**Comment by phavekes** [comment]
Until this is working, a logout - login will accomplish in effect the same as the SRAA will be prompted to choose an Institution

 (Daan van Renterghem - Feb 2, 2015)

**Comment by phavekes** [comment]
perhaps some reference http://stackoverflow.com/questions/19713802/symfony-2-3-how-do-i-refresh-the-authenticated-user-from-the-database
 (Daan van Renterghem - Feb 2, 2015)

**Comment by phavekes** [comment]
nice to have? Duplicate with #87412402? (Eefje van der Harst - Mar 26, 2015)

**Comment by phavekes** [comment]
@eefjev #87412402 ensures a list of institutions is available through the Middleware API, which is required to complete this feature (Reinier Kip - Mar 26, 2015)
